packages/tempus/src/Thyra_MultiVectorLinearOpWithSolveFactory.hpp

```diff
@@ -37,6 +37,8 @@ class MultiVectorLinearOpWithSolveFactory
    *
    * \param lowsf [in,persisting] The LOWSFB object that will be used to
    * create the LOWSB object for the diagonal blocks.
+   * \param multiVecRange [?] Description.
+   * \param multiVecDomain [?] Description.
```

Comment:
You'll need to fill out this documentation if you want to avoid the "undocumented parameter" warnings.  Or perhaps @etphipp needs to fill these in.